packages/core/package.json

```diff
@@ -19,16 +20,17 @@
     "@uniswap/v2-periphery": "1.1.0-beta.0",
     "@uniswap/v3-core": "^1.0.0-rc.2",
     "@uniswap/v3-periphery": "^1.0.0-beta.23",
-    "decimal.js": "^10.2.1"
+    "decimal.js": "^10.2.1",
+    "ethereum-waffle": "^3.4.0"
```

Comment:
we need waffle to do assertions using ethers in mocha such as if a contract reverts or to check for emitted events.